src/binaries/building-nym.md

```diff
@@ -18,6 +18,12 @@ sudo apt update
 sudo apt install pkg-config build-essential libssl-dev curl jq git
 ```
 
+- (Mac OS X) `pkg-config` , `brew`, `openss1`, `protobuf`, `curl`, `git`
+
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
does this install all of the above dependencies, or just brew?

Reply:
also if you're expecting people to run a curl script I suggest that you contextualise what it is and what it will do

Reply:
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" gets you all of those. curl, protobuf, pkg-config, git and openssl